### PR TITLE
dependencies/dependabot configuration

### PR DESCRIPTION
**Pull request by jotolo**
#### Description

Setting up dependabot could keep the libraries up to date.
This PR sets up dependabot to keep dependencies up to date

### CODE REVIEW

**Comment by jotolo** [comment]
Hi @MaicolBen Maybe we could set up dependabot to manage dependencies?
Let me know. Thanks in advance

**Comment by MaicolBen** [comment]
@jotolo sure, I asked @lynndylanhurley and enabled it, can you rebase the PR so we can fix the dependabot config file?

**Comment by jotolo** [comment]
@MaicolBen I kept the weekly schedule from master branch but maybe we could consider changing it to monthly schedule. 
We don't want PRs for libraries pilling up but it is your call . Let me know.

File: .github/dependabot.yml [hunk 1]
```diff
@@ -5,7 +5,9 @@
 
 version: 2
 updates:
-  - package-ecosystem: "" # See documentation for possible values
-    directory: "/" # Location of package manifests
+  - package-ecosystem: "bundler"
+    directory: "/"
     schedule:
       interval: "weekly"
```
Comment (MaicolBen) [inline .github/dependabot.yml:11]:
```suggestion
      interval: "monthly"
```

**Review by MaicolBen — APPROVED** [review]

**Comment by MaicolBen** [comment]
you're right, changed to monthly

**Pull request merged**